Doc/library/curses.panel.rst

```diff
@@ -6,6 +6,8 @@
 
 .. sectionauthor:: A.M. Kuchling <amk@amk.ca>
 
+**Source code:** :source:`Lib/curses/panel.py`
```

Comment:
This one is not really useful.

Reply:
@remilapeyre  Has it been deprecated?

Reply:
Not, it's just that the file is a wrapper for the C module: https://github.com/python/cpython/blob/master/Lib/curses/panel.py